docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3.7"
 services:
+  bark:
+    build:
+      dockerfile: Dockerfile
+    image: 'bark'
```

Comment:
We can use the `techrail/bark:0.1` image for now.

Reply:
Done.